src/dali/transaction_resolver.py

```diff
@@ -151,7 +166,34 @@ def resolve_transactions(
         if not isinstance(transaction, AbstractTransaction):
             raise Exception(f"Internal error: Parameter 'transaction' is not a subclass of AbstractTransaction. {transaction}")
 
+        if isinstance(transaction, IntraTransaction) and is_unknown(transaction.to_exchange):
+            # transfer out to unknown destination, record out address and
+            # amount for matching
+            if transaction.from_exchange == _COINBASE:
+                raw_data = json.loads(transaction.raw_data)
+                to_address = raw_data[_TO][_ADDRESS]
```

Comment:
This breaks abstraction layers: the transaction resolver should not have plugin-specific code.